core/impl-base/src/main/java/org/jboss/arquillian/core/impl/ManagerImpl.java

```diff
@@ -597,6 +599,27 @@ void debug(ObserverMethod method, boolean interceptor)
       }
    }
 
+   private void debugExtension(Class<?> extension)
```

Comment:
But debug/trace log is more than just sysout here - and this Manager is becoming a god class. I think it makes sense to split responsibilities.